### PR TITLE
Add Aurora Global Database Serverless V2 CDK Sample

### PR DESCRIPTION
**Pull request by nihilg**
*Description of changes:*
The PR contains a CDK sample application that creates Amazon Aurora Global database cluster across a primary and secondary region for SaaS applications that need global footprint and for disaster recovery strategies. The stack also includes a Fargate container application to test the primary and secondary regions with CRUD API operations. 

1. Added CDK code under /samples
2. Includes cdk-nag unit tests 
3. Updated README.md at the root to reference the new sample

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: samples/aurora-serverless-global-db-cdk/README.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    Expected Response : 
+    [{"id":1,"name":"Task1","status":"created"}]
+
+4. Attempt to write a task into the Secondary cluster 
+    ```
+    curl -X POST secondary-test-app.FargateServiceURL/tasks -H 'Content-Type: application/json' -d '{"name":"Task1","status":"created"}'
+    ```
+    Expected Response : error : cannot execute INSERT in a read-only transaction
+    
+    Note : You can enable Write forwarding feature to continue to use the Secondary cluster endpoint for write transactions as well.
```
Comment (jhart0) [inline samples/aurora-serverless-global-db-cdk/README.md:90]: Add link to docs for the write forwarding feature

**Review by jhart0 — CHANGES_REQUESTED** [review]
This is a great feature, thanks for contributing it! Just a few minor tweaks then it's good to go!

File: samples/aurora-serverless-global-db-cdk/cdk.context.json
[the remark this answers is not in the document]
Comment (nihilg) [reply samples/aurora-serverless-global-db-cdk/cdk.context.json:1]: Removed the generated content and left the file to be used to input ingressIpAddress

File: samples/aurora-serverless-global-db-cdk/lib/fargate-test-app-stack.ts [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+      allowAllOutbound: true,
+      vpc: props.vpc,
+    });
+
+    dbSecurityGroup.addIngressRule(
+      appSecurityGroup, Port.tcp(1234), 'allow port 1234 from the appSecurityGroup'
+    );
+
+    appSecurityGroup.addIngressRule(
+      Peer.ipv4('0.0.0.0/0'), Port.tcp(80), 'allow port 80'
```
[the remark this answers is not in the document]
Comment (nihilg) [reply samples/aurora-serverless-global-db-cdk/lib/fargate-test-app-stack.ts:26]: Yes, updated code to input the ingressIpAddress from cdk.context.json

File: samples/aurora-serverless-global-db-cdk/README.md (hunk 1 above)
Comment (nihilg) [reply samples/aurora-serverless-global-db-cdk/README.md:90]: Added

**Comment by jhart0** [comment]
Great, looks good to me!

**Pull request merged**